parsely/build.gradle

```diff
@@ -27,27 +29,38 @@ android {
             withJavadocJar()
         }
     }
+
+    testOptions {
+        unitTests {
+            includeAndroidResources = true
+        }
+    }
+
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_11.toString()
```

Comment:
Suggestion (💡): I suggests using `VERSION_1_8` for now, just like we are doing across every other lib and client repo (except from [DayOne-Android](https://github.com/search?q=repo%3Abloom%2FDayOne-Android%20JavaVersion.VERSION_11&type=code) 🤷), just to be on the safe side... 🤔

Reply:
I removed the explicit declaration as it's not really needed - it defaults to JVM 1.8

Reply:
Got it, thanks for doing that! 👍